### PR TITLE
Fix bug breaking owners bot

### PR DESCRIPTION
**Pull request by rcebulko**
This seems like it should never have worked, and yet it's been running for 2 months. My best guess is that when I added in cache invalidation/refreshing logic, it was broken, but it never had occasion to trigger until a recent OWNERS file change or deletion, and once that code path finally ran it broke.

### CODE REVIEW

File: owners/src/repo/virtual_repo.js [hunk 1]
```diff
@@ -143,11 +143,12 @@ module.exports = class VirtualRepository extends Repository {
     return await this.cache.readFile(repoPath, async () => {
       let contents = '';
       try {
-        contents = await this.github.getFileContents(relativePath);
+        const result = await this.github.getFileContents(relativePath);
```
Comment (jridgewell) [inline owners/src/repo/virtual_repo.js:146]: Nit: destructure

File: owners/src/api/github.js [hunk 2]
```diff
@@ -418,14 +418,18 @@ class GitHub {
   async getFileContents(filename) {
```
Comment (jridgewell) [inline owners/src/api/github.js:418]: The `@return` type is incorrect.

File: owners/src/repo/virtual_repo.js (hunk 1 above)
Comment (rcebulko) [reply owners/src/repo/virtual_repo.js:146]: `contents` needs to exist outside of the try/catch scope. Is it possible to destructure into an already-defined variable?

File: owners/src/api/github.js (hunk 2 above)
Comment (rcebulko) [reply owners/src/api/github.js:418]: done

File: owners/src/repo/virtual_repo.js (hunk 1 above)
Comment (jridgewell) [reply owners/src/repo/virtual_repo.js:146]: Technically yes, but it requires you to early declare the `sha` variable too:

```
let sha;
({ contents, sha} = await this.github.getFileContents(relativePath);
```

I find this ugly, so we can skip.

**Review by jridgewell — APPROVED** [review]

**Pull request merged**